test_apis/test_merchant.py

```diff
@@ -0,0 +1,206 @@
+from datetime import datetime
+import requests
+from requests.auth import HTTPBasicAuth
+from fake_useragent import UserAgent
+from .global_variables import *
+from .compare_json import compare_data_dictionaries
+
+
+class TestMerchantEndPoints(object):
+
+    url_merchants = MERCHANTS #for merchants
+    url_empty_search_scope = EMPTY_SEARCH_SCOPE_MODE
+    url_merchant_search_scope = MERCHANT_SEARCH_SCOPE_MODE
+    url_merchant_names = MERCHAT_NAMES
+    url_merchant_name = MERCHAT_NAME
+    url_merchant_with_id = MERCHANT_WITH_ID
+    url_merchant_outlets_with_id = MERCHANT_OUTLETS_WITH_ID
+    name_merchant_outlets_with_id = 'merchant_outlets_with_id'
```

Comment:
You are using name_ prefix for each of these. It can simply be concatenated with string.